doc/README.MacOSX

```diff
@@ -1,38 +1,54 @@
-mosh on Mac OS X
+mosh on macOS
 ================
 
 1. Install
 ----------
 
-(Homebrew includes mosh as a fomula. Say "brew install mosh" to install it.)
-
 To install this source package:
 
-- Install following using MacPorts:
-  oniguruma5
+- Install following using homebrea package:
+  oniguruma
   gmp
-- M1 Macs are experimentaly supported on 0.2.8_RC3:
+- M1 Macs are experimentaly supported on 0.2.8_RC3 or later.:
+
 ```
 export CPATH=/opt/homebrew/include
 export LIBLARY_PATH=/opt/homebrew/lib
 ./configure 
 make
 make test
 ```
+
+2. Limitations
+--------------
+
+- (10.5 and maybe 10.6): Configure script may fail to recognise target 
+  architecture. set CC=i686-darwin10-gcc CXX=i686-darwin10-c++ or so 
+  to avoid this problem.
+- CMake builds does not support FFI callbacks.
+
```

Comment:
I think it's okay to remove any mention of now-unsupported versions. Homebrew requires 10.15 or later now.

Reply:
Makes sense. Removed them.